### DISCUSSION TITLE
Possible Typo in Documentation? "talosctcl" -> "talosctl"

### DISCUSSION THREAD

**Discussion by Gamedirection** [siderolabs/talos#8718] [General]
https://www.talos.dev/v1.7/talos-guides/install/local-platforms/docker/#create-the-cluster

Under Creating a Cluster:
```
$ **talosctcl** config info
...
Endpoints:           127.0.0.1:38423
```
talosctcl should be talosctl unless I am mistaken? 

Awesome work!

**Comment by steverfrancis** [comment]
Thanks - fixed in the 1.8 docs.
